docs/common/guidelines/1226-coding-style.md

```diff
@@ -1,6 +1,5 @@
 ---
 slug: /1226/coding-style
-displayed_sidebar: "0.2"
```

Comment:
@gerhard per our discussion today, removal of this field in common files does not appear to affect the render. The files continue to be referenced in the sidebar configuration.